.circleci/config.yml

```diff
@@ -29,6 +29,7 @@ jobs:
       - image: olizilla/ipfs-dns-deploy:latest
         environment:
           DOMAIN: webui.ipfs.io
+          DEV_DOMAIN: dev.webui.ipfs.io
```

Comment:
```suggestion
```

Not needed... The domain here is actually the "zone". The zone is a logical grouping of dns records that can include subdomains of the root domain. The zone here is `webui.ipfs.io`, and we'll use that as the root for configuring `_dnslink.webui.ipfs.io` and `_dnslink.dev.webui.ipfs.io`